client/my-sites/email/titan-new-mailbox-list/index.jsx

```diff
@@ -9,14 +9,18 @@ import {
 	sanitizeEmailSuggestion,
 	validateMailboxes,
 } from 'calypso/lib/titan/new-mailbox';
-import TitanNewMailbox from 'calypso/my-sites/email/titan-new-mailbox';
+import TitanNewMailbox, {
+	FULL_NAME_TITAN_FIELD,
+	PASSWORD_RESET_TITAN_FIELD,
```

Comment:
Maybe start these constants with `TITAN_` to make them more clearly related? e.g. `TITAN_FULL_NAME_FIELD` and `TITAN_PASSWORD_RESET_FIELD`.